routesrv/routesrv_test.go

```diff
@@ -196,6 +213,30 @@ func TestFetchedRoutesAreServedInEskipFormat(t *testing.T) {
 	wantHTTPCode(t, w, http.StatusOK)
 }
 
+func TestRedisIPs(t *testing.T) {
+	defer tl.Reset()
+	ks, _ := newKubeServer(t, loadKubeYAML(t, "testdata/redis.yaml"))
+	ks.Start()
+	defer ks.Close()
+	rs := newRouteServerWithOptions(t, routesrv.Options{
+		SourcePollTimeout:               pollInterval,
+		KubernetesURL:                   ks.URL,
+		KubernetesRedisServiceNamespace: "namespace1",
+		KubernetesRedisServiceName:      "service1",
+	})
+
+	w := getRedisURLs(rs)
+
+	want := parseRedisIP(t, "testdata/redis-ip.txt")
+	got := w.Body.Bytes()
+
+	if !bytes.Equal(got, want) {
```

Comment:
I don't know but if this test is flaky we need to sort the string values by line to compare

Reply:
I can sort, but I don't think it's flaky. I need to check how the IPs are returned from k8s dataclient first.

Reply:
just run something like:
```
go test -run MYTest -count 10 ./routesrv
```